db/migrations/1646958763396-Item.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Item1646958763396 implements MigrationInterface {
+  name = 'Item1646958763396';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Create a foreing key that relates this table with the company table

Reply:
👌